rs/ethereum/ledger-suite-orchestrator/src/scheduler/mod.rs

```diff
@@ -898,6 +899,7 @@ fn icrc1_ledger_init_arg(
     archive_controller_id: PrincipalId,
     archive_more_controller_ids: Vec<PrincipalId>,
     cycles_for_archive_creation: Nat,
+    index_principal: Option<Principal>,
```

Comment:
nit: we should not call that method with `None` as `index_principal` so I would suggest to modify the method signature as follows:
```suggestion
    index_principal: Principal,
```

Reply:
Fixed!